iphone/cli/commands/_build.js

```diff
@@ -1160,15 +1171,19 @@ iOSBuilder.prototype.configOptionPPuuid = function configOptionPPuuid(order) {
 					return callback(new Error(__('Specified provisioning profile UUID "%s" is expired', value)));
 				}
 
-				let cert;
+				let certs;
 				if (target === 'device') {
-					cert = _t.findCertificate(cli.argv['developer-name'], 'developer');
+					certs = _t.findCertificate(cli.argv['developer-name'], 'developer');
 				} else {
-					cert = _t.findCertificate(cli.argv['distribution-name'], 'distribution');
+					certs = _t.findCertificate(cli.argv['distribution-name'], 'distribution');
 				}
 
-				if (cert && p.certs.indexOf(cert.pem.replace(pemCertRegExp, '')) === -1) {
-					return callback(new Error(__('Specified provisioning profile UUID "%s" does not include the "%s" certificate', value, cert.name)));
+				const pems = certs.map(function (c) {
+					return c.pem.replace(pemCertRegExp, '');
+				})
+
+				if (certs.length > 0 && intersection(p.certs, pems).length < 0) {
```

Comment:
If the intersection returns 0, it is okay?

Reply:
fixed.